components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -3798,7 +3801,7 @@ function doValidation() {
             <td>
                 <div class="sectionCheckbox">
                     <input id="logoutRequestSigned" name="logoutRequestSigned"
-                           type="checkbox" <%=logoutRequestSignedChecked%>/>
+                           type="checkbox" <%=Encode.forHtmlAttribute(logoutRequestSignedChecked)%>/>
```

Comment:
The variable value is hardcoded. Further this is not an attribute. The variable fully represents the attribute key and value here. So no need to encode.